### ISSUE TITLE
Need clarification in parse_test_data function

### ISSUE THREAD

**Issue by ranjeetkumar** [iit-cs579/main#251]
I have two questions in this.

1) What is mean by this statement "Note that vocab should
    be passed to the vectorize function to ensure the feature
    mapping is consistent from training to testing"  What to do when it is found inconsistent?
2) path variable that to be passed from this function to read_data should be hard coded?

**Comment by aronwc** [comment]
1) By this I mean that the same vocabulary object should be used to vectorize the training and testing data. E.g., if the word "dog" maps to index 10 in the training set, then it should also do so in the test set.

2) The path can be hard coded for the training/test data. The main method has an example
>  docs, labels = read_data(os.path.join('data', 'train'))

**Comment by ranjeetkumar** [comment]
Thanks a lot!
